webpack.config.js

```diff
@@ -29,23 +30,15 @@ module.exports = (env, argv) => {
       headers: { "Access-Control-Allow-Origin": "*" },
     },
     context: __dirname,
-    entry: ["./frontend/js/index.js"],
+    entry: ["./frontend/js/index.tsx"],
     output: isDev ? localhostOutput : productionOutput,
     module: {
       rules: [
         {
-          test: /\.[jt]sx?$/,
-          exclude: nodeModulesDir,
-          use: [
-            {
-              loader: require.resolve("babel-loader"),
-              options: {
-                plugins: [
-                  isDev && require.resolve("react-refresh/babel"),
-                ].filter(Boolean),
-              },
-            },
-          ],
+          test: /\.(js|mjs|jsx|ts|tsx)$/,
```

Comment:
Hmm, since we're targeting projects for modern environments, such as updated browsers and newer versions of Node.js, I feel we can reduce the initial setup while keeping the boilerplate focused on modern module formats like ECMAScript Modules. This eliminates the need to include support for CommonJS (CJS) in the regex.

Reply:
Sounds good.